docs/details/other-components/obj_id.rst

```diff
@@ -33,26 +33,26 @@ Below APIs needed to be implemented and linked to lvgl.
 
 .. code-block:: c
 
-    void lv_obj_set_id(lv_obj_t * obj, void * id);
-    void lv_obj_assign_id(const lv_obj_class_t * class_p, lv_obj_t * obj);
-    void lv_obj_free_id(lv_obj_t * obj);
-    const char * lv_obj_stringify_id(lv_obj_t * obj, char * buf, uint32_t len);
+    void lv_obj_set_id(lv_obj_t * widget, void * id);
```

Comment:
Is this change intentional? It's inconsistent with source code.

Reply:
> Is this change intentional? It's inconsistent with source code.

Yes, it was intentional and coordinated with Gábor.  Reason:  we are both soon expecting the type name `lv_obj_t` to change to `lv_widget_t` or something close to that.  Gábor gave me the "green light" to start using the term "widget" in this context, including in function arguments.

Reply:
Hi @kisvegabor , 
Could you confirm? When we upgrade to lvgl v9, we agreed to not mess with `lv_obj_` prefix, in order to avoid massive API break.
cc @FASTSHIFT @W-Mai @terry0012

Reply:
Confirmed. 

> When we upgrade to lvgl v9, we agreed to not mess with lv_obj_ prefix, in order to avoid massive API break.

IIRC, we agreed to not do it in v9. :smile: IMO we should slowly transition towards the `widget`name and we can do it without API breaks by adding backward compatibility defines in `lv_api_map.h`.

I don't have an exact plan when and how to introduce this refactoring, but once the docs were reworked, I find it a good time to start using the "widget" name.

Reply:
I think lv_view_t is more shorter 🤔

Reply:
I suggest launching a public vote: choose to keep `obj` or use `widget`, because LVGL has such a large audience, and this core API change will affect everyone.